scripts/Jenkinsfile

```diff
@@ -10,111 +10,111 @@ node {
   def testContainer = "${job_name}${env.BUILD_NUMBER}_test"
   def imageName = "acceptance-tests-${job_name}${env.BUILD_NUMBER}"
   def envFile = ".env"
+  def qaseEnvFile = ".qase.env"
```

Comment:
Do we need a separate env file for Qase? can't we use the same .env file?

Reply:
both..i just prefer to separate , to be cleaner

Reply:
Can we have an example for this file?